schema.graphql

```diff
@@ -94,6 +94,7 @@ scalar Any
 type AppUserProfile {
   _id: ID!
   adminFor: [Organization]
+  adminApproved: Boolean!
```

Comment:
`adminApproved` needs to be on a per organization basis. You'll need to change the schema to be similar to that of `adminFor`

Reply:
Okay I will make that change for `adminApproved` to be like `adminFor`.